lib/pure/asynchttpserver.nim

```diff
@@ -46,7 +94,7 @@ const
 
 when (NimMajor, NimMinor) >= (1, 1):
   const
-    chunkSize = 8*1048 ## This seems perfectly reasonable for default chunkSize.
+    chunkSize = 8*1024 ## This seems perfectly reasonable for default chunkSize.
```

Comment:
Why modify this?

Reply:
When I submit the first pull request I wrote the wrong number. I put 1048 instead of 1024. I need to correct it because the first pull request has already been approved.